### ISSUE TITLE
[4.x] Can't remove tags with Taggable Fieldtype

### ISSUE THREAD

**Issue by duncanmcclean** [statamic/cms#7814]
### Bug description

I've noticed that in 4.0, you can't currently remove tags in the Taggable fieldtype. It just won't let you.

### How to reproduce

1. Add the 'Taggable' fieldtype to a blueprint
2. Go into an entry with that blueprint
3. Add some tags
4. Try and remove them using the 'x' button
5. They don't get removed

Loom: https://www.loom.com/share/718e61dfc8ad4bd5aa90cf242453b63f

### Environment

```yaml
Environment
Application Name: Statamic
Laravel Version: 9.52.5
PHP Version: 8.2.3
Composer Version: 2.2.6
Environment: local
Debug Mode: ENABLED
URL: 2023-sc-sandbox.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Simple Commerce
Currencies: GBP
Gateways: Stripe, Mollie, PayPal, Dummy
Repository: Customer: DoubleThreeDigital\SimpleCommerce\Customers\EntryCustomerRepository
Repository: Order: DoubleThreeDigital\SimpleCommerce\Orders\EntryOrderRepository
Repository: Product: DoubleThreeDigital\SimpleCommerce\Products\EntryProductRepository
Shipping Methods: Free Shipping
Tax Engine: DoubleThreeDigital\SimpleCommerce\Tax\BasicTaxEngine

Statamic
Addons: 2
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: dev-master (as of 1st April 2023)

Statamic Addons
doublethreedigital/runway: 4.0.0
doublethreedigital/simple-commerce: 5.x (unreleased)
```

### Installation

Starter Kit using via CLI

### Antlers Parser

runtime (new)

**Comment by jackmcdade** [comment]
This appears to be some kind of collision with Sortable. 🤔

**Comment by jasonvarga** [comment]
Fixed by #7824
